aws-rds-dbcluster/aws-rds-dbcluster.json

```diff
@@ -379,6 +383,7 @@
     "/properties/DBClusterIdentifier"
   ],
   "writeOnlyProperties": [
+    "/properties/DBInstanceParameterGroupName",
```

Comment:
AFAIU, Write only parameters will not return in read handler which means customer will not see the drift of `DBInstanceParameterGroupName` when he run `Detect Drift` action on stack although template value does not match physical value. Should we support it in Read handler or there is strong reason behind it?

Reply:
This is an intended behaviour. It is not possible to read this parameter from DBCluster, it is only meant to be supplied during Major Version Upgrade. Therefore, it is made a write-only parameter.

Reply:
But we can retrieve it from member DBInstance. If is upgraded manually out of band but stack still have old value. Customer run `Detect drift`  and it doesn't appear but when customer execute stack it will fail when trying to apply this parameter.
This case can happen as we did not support this feature before and customers might have scripts that do this manually.